tap_messagebird/streams.py

```diff
@@ -84,20 +127,8 @@ class ConversationMessagesStream(MessagebirdConversations):
     # Optionally, you may also use `schema_filepath` in place of `schema`:
     schema_filepath = SCHEMAS_DIR / "conversation_message.json"
     parent_stream_type = ConversationsStream
-
-    def get_url_params(
```

Comment:
From looking at produciton data, most conversations don't have a ton of back and forth so also keeping a state tracker for number of messages is probably overboard. We could but I wanted to get rid of the complexity and potential for something messing up